BigriverBookstore-apple/Sources/Controller/MainNavigationViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+import Cocoa
+//  MainNavigationViewController.swift
+//  BigriverBookstore-apple
+//
+//  Created by Viorel Porumbescu on 07/03/2019.
+//  Copyright © 2019 AgileFreaks. All rights reserved.
+//
+import Foundation
+
+class MainNavigationViewController: NSViewController {
+    @IBOutlet var previousButton: NSButton!
+    @IBOutlet var currenSectionTitleTextField: NSTextField!
+    
+    private var sectionsNames: [String] = [""]
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.previousButton.isHidden = true
+        self.currenSectionTitleTextField.stringValue = ""
+    }
+    
+    @IBAction func goBackInNavigationStack(_ sender: Any) {
+        self.previousButton.animator().isHidden = children.count == 2
```

Comment:
@c-Viorel I'm lost here :) Let's discuss offline on what you wanted to achieve here, maybe we have a better way to do it.